INSTALL.md

```diff
@@ -46,7 +50,7 @@ sudo docker run -it \
 ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
-2\. Install the dependencies (currently only for Python2):
+2\. Install the dependencies (currently only for Python2):  
```

Comment:
Did you add whitespace on purpose?

Reply:
This was intentional. The TODO was added to the line below but Markdown requires the two spaces in the previous line to actually make it so. This was just a formatting thing.